docs/source/getting-started/statistical-modeling/regression.ipynb

```diff
@@ -0,0 +1,557 @@
+    "\n",
+    "The parameter `runs` controls the number of independent Theil-Sen calculations performed and stacked together. The default is 1. Increasing `runs` can improve the robustness and accuracy of the results; however, it can also increase computational cost. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def make_theil_sen_percentiles(runs: int=1):\n",
+    "\n",
+    "    return dp.t.make_user_transformation(\n",
+    "        # Defines type of input. \n",
+    "        input_domain=dp.np_array2_domain(T=float, num_columns=2),\n",
+    "\n",
+    "        # The symmetric distance metric measures the number of changes required for transformation.\n",
+    "        input_metric=dp.symmetric_distance(),\n",
+    "\n",
+    "        # Define output format. \n",
+    "        output_domain=dp.np_array2_domain(T=float, num_columns=2),\n",
+    "\n",
+    "        # Define output metric. \n",
+    "        output_metric=dp.symmetric_distance(),\n",
+    "\n",
+    "        # Apply the function runs times,\n",
+    "        function=lambda x: np.vstack([compute_points_on_percentiles(x) for _ in range(runs)]),\n",
+    "        stability_map=lambda b_in: runs * b_in)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 4. Create a Function to Select a Specific Column"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This function creates a transformation to select a column from a dataset. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def make_select_column(j):\n",
+    "    return dp.t.make_user_transformation(\n",
+    "        input_domain=dp.np_array2_domain(T=float, num_columns=2),\n",
+    "        input_metric=dp.symmetric_distance(),\n",
+    "        output_domain=dp.vector_domain(dp.atom_domain(T=float)),\n",
+    "        output_metric=dp.symmetric_distance(),\n",
+    "        function=lambda x: x[:, j],\n",
+    "        stability_map=lambda b_in: b_in)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 5. Create a Function to Make Private Quantiles in Bounds"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this section, we will chain a transformation and measurement to compute the median with the existing quantile scores methods. First, we define the `discrete_scores` transformation by calling `dp.t.make_quantile_score_candidates` which outputs scores for how similar each candidate is to the given alpha-quantile on the input dataset."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "Transformation(\n",
```

Comment:
Is this also a part of the code? Or is this some output? Might want to enclose it in a box or something.